src/robot/output/listeners.py

```diff
@@ -152,9 +152,8 @@ def _get_version(self, listener):
             if version not in (2, 3):
                 raise ValueError
         except AttributeError:
-            raise DataError("Listener '%s' does not have mandatory "
-                            "'ROBOT_LISTENER_API_VERSION' attribute."
-                            % self.name)
+            "Sicne 7.1, Listener API version is by default considered to be V3 unless explicitly mentioned as V2"
```

Comment:
No need for this kind of a comment. If it was needed, it should be a normal comment starting with a `#`.